src/tools/create-require.js

```diff
@@ -0,0 +1,21 @@
+'use-strict';
+
+const { dependencies, devDependencies } = require('../../package.json');
+const { RequireModuleError } = require('../errors');
+
+const createRequire = () => module => {
+  const libs = Object.keys(dependencies).concat(Object.keys(devDependencies));
+  const allowedLibs = ['lodash'];
+  const disallowedModules = ['fs', 'child_process'];
+  const disallowedLibs = libs.filter(d => !allowedLibs.includes(d));
+
+  if (disallowedModules.includes(module) || disallowedLibs.includes(module)) {
+    throw new RequireModuleError(
+      `You are attempting to use a disallowed module: "${module}"`
```

Comment:
I would change this error to `..attempting to require a disallowed` since it pinpoints the source of the error at the `require` call

Reply:
I think we can make this moot by just going right to `require`